### PR TITLE
Update alert-policies.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the permissions needed in Azure AD as well 

Reference https://learn.microsoft.com/en-us/microsoft-365/compliance/compliance-manager-alert-policies?view=o365-worldwide#alert-policy-permissions

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9985

File: microsoft-365/compliance/alert-policies.md [hunk 1]
```diff
@@ -44,7 +44,10 @@ Here's a quick overview of how alert policies work and the alerts that are trigg
 
 1. An admin in your organization creates, configures, and turns on an alert policy by using the **Alert policies** page in the compliance portal or the Microsoft 365 Defender portal. You can also create alert policies by using the [New-ProtectionAlert](/powershell/module/exchange/new-protectionalert) cmdlet in Security & Compliance PowerShell.
 
-   To create alert policies, you have to be assigned the Manage Alerts role or the Organization Configuration role in the compliance portal or the Defender portal.
+ In addition to holding a Compliance Manager role, users also need an Azure AD role as follows:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/alert-policies.md:47]:
```suggestion
   In addition to holding a Compliance Manager role, users also need an Azure AD role as follows:

```

File: microsoft-365/compliance/alert-policies.md [hunk 2]
```diff
@@ -44,7 +44,10 @@ Here's a quick overview of how alert policies work and the alerts that are trigg
...
+
+To view alerts and alert policies: the Security reader role in Azure AD
+To create or update alert policies: the Compliance administrator, Compliance data administrator, Security administrator, or Security operator role in Azure AD
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/alert-policies.md]:
```suggestion
   - To view alerts and alert policies: the Security reader role in Azure AD.
   - To create or update alert policies: the Compliance administrator, Compliance data administrator, Security administrator, or Security operator role in Azure AD.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@robmazz Gentle reminder

**Comment by yogkumgit** [comment]
Closing as per the comment in #9985

**Pull request closed without merging**